src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -88,6 +87,32 @@ const MessageDetails: FC<MessageDetailsProps> = ({users, children, message, doma
   );
 };
 
+const UnreachableBackendMessageDetails: FC<MessageDetailsProps> = props => {
```

Comment:
If I good remember we should pass types to props. 🤔 
```suggestion
const UnreachableBackendMessageDetails = (props: MessageDetailsProps) => {
```

Reply:
That's also fine, yes. Either way work for me

Reply:
fixed https://github.com/wireapp/wire-webapp/pull/15854/commits/c406992bab139bfffc28386c658b6fa71dbf219d